spec.md

```diff
@@ -8,66 +8,96 @@ title: Semantic Specification
 
 
 ~~~
-Parentage/causal relationships for spans from a single trace
+Causal relationships for Spans in a single trace
 
-        [Span A]  <--(the root span)
+        [Span A]  ←←←(the root span)
             |
      +------+------+
      |             |
- [Span B]      [Span C]
+ [Span B]      [Span C] ←←←(Span A is Span C's parent)
      |             |
- [Span D]      +---+-----+
-               |         |
-           [Span E]  [Span F]
-                         |
-                +--------+--------+
-                |        |        |
-            [Span G] [Span H] [Span I]
+ [Span D]      +---+-------+
+               |           |
+           [Span E]    [Span F] >>> [Span G] >>> [Span H]
+                                       ↑
+                                       ↑
+                                       ↑
+              (Span F is Span G's "Sequence Predecessor")
 
 ~~~
 
 ~~~
-Temporal relationships for spans from a single trace
+Temporal relationships for all of the above
 
 ––|–––––––|–––––––|–––––––|–––––––|–––––––|–––––––|–––––––|–> time
 
  [Span A···················································]
    [Span B··············································]
       [Span D··········································]
     [Span C········································]
-         [Span E···········]   [Span F········]
-                                [Span G······]
-                                [Span H··]
-                                        [Span I·]
+         [Span E·······]        [Span F··] [Span G··] [Span H··]
 ~~~
 
 ## Traces
 
-A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system. A Trace can be thought of as a tree of Spans. (See the ASCII diagrams above)
+A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system. A Trace can be thought of as a DAG of Spans. (See the ASCII diagrams above)
```

Comment:
Would it be clearer if you said directed acyclic graph instead of using the acronym? It seems nitpicking, but not everyone in this document's target audience may have a comfortable grasp of graph theory concepts.